src/makeCalculator.js

```diff
@@ -4,7 +4,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    // Метод для додавання
+    add: function (a) {
+      this.result += a;
+    },
+
+    // Метод для віднімання
+    subtract: function (a) {
```

Comment:
[CODE KNOWLEDGE] - Use the method shorthand syntax for the `subtract` method as well. Change `subtract: function (a) {` to `subtract(a) {`.